README.md

```diff
@@ -32,6 +32,13 @@ Run the index.html file in your favorite browser
 - Twitter: [@MohamedJadib](https://twitter.com/MohamedJadib)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamed-jadib-942a5041/)
 
```

Comment:
- [ ] [OPTIONAL] It would have been nice to add how to run tests locally, having already added tests.